src/main/drivers/pwm_output.c

```diff
@@ -352,6 +353,28 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         }
     }
 }
+
+void pwmWriteDshotCommandAll(uint8_t motorCount, uint8_t command)
+{
+    if (command <= DSHOT_MAX_COMMAND) {
```

Comment:
It's more about maintainability, and trying to implement function only once, to avoid the situation where one code path gets updated / fixed, and another one is forgotten.